pyi.py

```diff
@@ -637,12 +640,15 @@ class UnionAnalysis(NamedTuple):
     multiple_literals_in_union: bool
     non_literals_in_union: bool
     combined_literal_members: list[_LiteralMember]
+    multiple_builtins_types_in_union: bool
+    non_builtins_types_in_union: bool
+    combined_builtins_types: list[_BuiltinsTypeMember]
 
 
 def _analyse_union(members: Sequence[ast.expr]) -> UnionAnalysis:
     """Return a tuple providing analysis of a given sequence of union members.
 
-    >>> union = _ast_node_for('Union[int, memoryview, memoryview, Literal["foo"], Literal[1]]')
+    >>> union = _ast_node_for('Union[int, memoryview, memoryview, Literal["foo"], Literal[1], type[float], type[str]]')
```

Comment:
This line now stretches to 120 characters -- let's do this instead:

```suggestion
    >>> union = _ast_node_for(
    ...     'Union[int, memoryview, memoryview, Literal["foo"], Literal[1], type[float], type[str]]'
    ... )
```